dsms/knowledge/utils.py

```diff
@@ -323,19 +323,15 @@ def _update_kitem(new_kitem: "KItem", old_kitem: "Dict[str, Any]") -> Response:
             "id",
             "kitem_apps",
             "created_at",
-            "external_links",
             "dataframe",
             "access_url",
         },
         exclude_none=True,
     )
     payload.update(
-        external_links={
-            link.label: str(link.url) for link in new_kitem.external_links
-        },
         **differences,
     )
-    if new_kitem.custom_properties:
+    if new_kitem.custom_properties is not None:
```

Comment:
Perhaps a comment would be useful to avoid the deletion of "is not None" in case we forget why it's needed.

Reply:
the behaviour has been updated. A dedicated field serializer for the `custom_properties`-field was introduced. In the future, the automated buffering will be removed. See #57